app/content-field-type-options/boolean/template.html

```diff
@@ -4,12 +4,40 @@
         <label class="form-check-label" for="required">Required</label>
     </div>
 
-    <div class="form-floating switch mb-3">
+    <div class="form-floating mb-3">
         <select name="control_type" id="control_type" class="form-select" ng-model="contentField.options.control_type">
             <option value="checkbox">Checkbox</option>
             <option value="switch">Switch</option>
+            <option value="radiobuttons">Radiobuttons</option>
         </select>
         <label class="form-label" for="control">Control Type</label>
     </div>
 
+    <div ng-if="contentField.options.control_type=='radiobuttons'">
+        <div class="form-floating mb-3">
+            <select name="label_set" id="label_set" class="form-select" ng-model="contentField.options.label_set">
+                <option value="true_false">True/False</option>
+                <option value="yes_no">Yes/No</option>
+                <option value="open_closed">Open/Closed</option>
```

Comment:
I get the idea but I think we should give users an option to just simply decide the labels they want to use for `True` and `False`, this will be very hard for localization and in some languages, it's hard to just match, like in french `Enabled` and `Active` don't really have two different words for... Also we gonna miss a lot of other words that could be assigned as true or false values, I can think of On/Off, Yin/Yang, Dark/Light, Enabled/Disablesd, Approved/Rejected and the list goes on and on.

Reply:
Agreed.  Perhaps this is handled via localization, or just provide a way to change the labels for the boolean?

Reply:
The "other" option allows them to put in whatever label values they want. Try it out. :)

Reply:
Yeah, it just seems a bit opinionated.  If we leave the labels configurable, then we don't have to support hard-coded label options that may or may not make sense for the use case and/or language.

Reply:
ah ok.  I can take out the label sets and just go straight to the "other" options.

Reply:
I suppose this is a nice compromise.  Does it require unnecessary complexity though, instead of just allowing the admin to change the default labels?

Reply:
pretty easy